MovieTime/MovieTime/Model/MTModelMovieDBConfiguration.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MTModelMovieDBConfiguration.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 10/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+struct MTModelMovieDBConfiguration: Codable {
+
+    var images: MTModelMovieDBConfigurationImages?
```

Comment:
what is the design pattern you've used here to justify separating into 2 different entities `MTModelMovieDBConfiguration` and `MTModelMovieDBConfigurationImages`?

Reply:
There are 2 classes for that because the service models are "copied" into the application. One for configuration and other for the nested object (images).
I could use the `CodingKey` protocol and the follow `KeyedEncodingContainer<K>` method, avoiding 2 different objects, but I believe the service models should be reflected within the application, making it easy to maintain and align with other platforms. Besides that, using `CodingKeys` increases the development time and, unfortunately, many times the time must be balanced with quality.
```
public mutating func nestedContainer<NestedKey>(keyedBy keyType: NestedKey.Type, forKey key: KeyedEncodingContainer.Key) -> KeyedEncodingContainer<NestedKey> where NestedKey : CodingKey
```

Reply:
having separate entities is indeed the best approach that I know so far ;)